setup.cfg

```
@@ -76,6 +76,7 @@ install_requires =
     # unpinned transient dependencies used for type checking
     rx
     promise
+    tomli>=2.*
```

(or something like this is the interface remains the same, note would require 3.11 testing in GH actions)

```suggestion
    tomli>=2.*; py<3.11
```

---

The change I'll be pushing up is based on https://peps.python.org/pep-0508/#environment-markers

```suggestion
    tomli>=2.*; python_version<"3.11"
```